src/utils/daos.c

```diff
@@ -704,6 +705,13 @@ common_op_parse_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 		}
 	}
 
+	/* unexpected extra arguments not allowed */
+	if (optind < argc - 2) {
```

Comment:
With the offset 2 being reused in a number of places, should probably be made into a const variable

Reply:
right, and this will allow to also make it more readable and understandable ! thx.